sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -354,6 +353,8 @@ class Analyzer(override val catalogManager: CatalogManager) extends RuleExecutor
       UpdateAttributeNullability),
     Batch("UDF", Once,
       ResolveEncodersInUDF),
+    Batch("RewriteMergeInto", Once,
```

Comment:
for safety, shall we use `fixedPoint`? The rule was in a `fixedPoint` batch before.

Reply:
Updated.